File: Nim/primes.nim
import strformat
from times import nil

let start = times.cpuTime()

var primes: seq[int] = @[]

for i in 0..<15000:
    var factors: seq[int] = @[]

    for j in 1..i:
        if i mod j == 0:
            factors.add(j)

    if len(factors) == 2:
        primes.add(i)

echo(primes)

echo(fmt"Time taken: {times.cpuTime()-start}s")

File: Python/primes.py
from time import time

start = time()

primes = []

for i in range(0, 15000):
    factors = []

    for j in range(1, i+1):
        if i % j == 0:
            factors.append(j)

    if len(factors) == 2:
        primes.append(i)

print(primes)

print(f"Time taken: {round(time()-start, 3)}s")
